app/code/community/Kirchbergerknorr/UrlCacheControl/Mage/Core/Model/Design/Package.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * @author      Nick Dilßner <nd@kirchbergerknorr.de>
+ * @copyright   Copyright (c) 2017 kirchbergerknorr GmbH (http://www.kirchbergerknorr.de)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class Kirchbergerknorr_UrlCacheControl_Mage_Core_Model_Design_Package extends Mage_Core_Model_Design_Package
+{
+    /**
+     * parameter _type
+     */
+    const SKIN = 'skin';
+
+    /**
+     * add modified timestamp to url
+     *
+     * {@inheritdoc}
+     */
+    public function getSkinUrl($file = null, array $params = array())
+    {
+        if (empty($params['_type'])) {
```

Comment:
Why do you need this here? The parent method sets this anyway, doesn't it?

Reply:
Yes, nur i need the path-to-file for the modified timestamp.

The Parameter `_type` ist required for `getFilename()` which Returns the Design path-to-file on existing File, Else it Returns Default path-to-file.

I will add a comment

Reply:
i added the copy comment liek in merged Css and Js, because it is more or less the copy, but optimized with getFilename, which does the same like the parent, except the parent need the `$param` reference of _fallback()